engine/installation/linux/docker-ce/debian.md

```diff
@@ -208,16 +208,15 @@ from the repository.
     ```bash
     $ apt-cache madison docker-ce
 
-    docker-ce | {{ minor-version }}.0~ce-0~debian-jessie | {{ download-url-base}} jessie/stable amd64 Packages
+    docker-ce | {{ minor-version }}.0~ce-0~debian | {{ download-url-base}} jessie/stable amd64 Packages
```

Comment:
@andrewhsu Is there a way to remove the suffix from older packages as well? The instructions show how to install the _latest_ version, but also explain people can pin to an older version, which won't work because those still have a suffix